tests/bootstrap.php

```diff
@@ -4,10 +4,14 @@
 
 use Symfony\Component\Process\Process;
 
+$minkTestServerPort = isset($_SERVER['WEB_FIXTURES_HOST'])
+    ? parse_url($_SERVER['WEB_FIXTURES_HOST'], PHP_URL_PORT)
+    : '8002';
+
 $minkTestServer = new Process([
     PHP_BINARY,
     '-S',
-    '0.0.0.0:8002',
+    "0.0.0.0:$minkTestServerPort",
```

Comment:
It works, but for performance reasons maybe it's better to be rewritten without double quote usage as:

```php
'0.0.0.0:' . $minkTestServerPort,
```

Reply:
It should be a well know fact that string interpolation in PHP has no performance impact and anyway that code executes once (it's in the bootstrap).

Anyway, I applied the change.

Reply:
Thank you.